editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -152,6 +158,13 @@ export const ComponentPicker = React.memo((props: ComponentPickerProps) => {
     [flatComponentsToShow, highlightedComponentKey, onItemClick, selectIndex],
   )
 
+  const categorizedComponents = [
+    {
+      label: 'Everything',
+      items: flatComponentsToShow,
+    },
+  ]
```

Comment:
To address @balazsbajorics feedback, it's actually not super simple to just nuke all categories because the data needs to be formatted in categories still for it to show up based on how it's rendered now.

Here, I am just using a single category now, and only showing the tabs when there are more than 1 category, so the component is now hidden.

Reply:
sounds like a good solution to me!